target_postgres/target_tools.py

```diff
@@ -91,6 +91,7 @@ def _line_handler(state_tracker, target, invalid_records_detect, invalid_records
                   max_batch_size, line):
     try:
         line_data = json.loads(line)
+        line_data[DATAMILL_RAW_LINE_SIZE] = len(line)
```

Comment:
Should we only add this for record types?

Reply:
Makes sense. Easy change to make.